### PR TITLE
Order allocation, approval, finance and cost-recovery rows desc by date

### LINKED ISSUE #38: Update to Ordering in Cost Recovery Table
Please update the ordering of records in the Cost Recovery Table to sort by the month, either in ascending or descending order. Currently, the records appear to be ordered by the date they were created, which is not as useful for users.

Additionally, could you format the values in the month column to the YYYY-MM format? This change will make it easier for users to clearly see the chronological order of the months.

### PR DESCRIPTION
**Pull request by Iain-S**
As the title suggest, we explicitly order the rows in the details page tables. This may also affect the CLI output but that's not a bad thing.

Closes #38

### CODE REVIEW

File: rctab/routers/accounting/routes.py [hunk 1]
```diff
@@ -391,60 +391,77 @@ def get_total_usage(
...
-            cost_recovery.c.subscription_id,
-            cost_recovery.c.finance_id,
-            cost_recovery.c.month,
-            cost_recovery.c.finance_code,
-            cost_recovery.c.amount,
-            cost_recovery.c.date_recovered,
-        ]
-    ).where(cost_recovery.c.subscription_id == sub_id)
+    """Get all cost recovery items for a subscription in asc date order."""
+    # order by month asc
```
Comment (dlpbc) [inline rctab/routers/accounting/routes.py:450]: Just double checking whether this comment aligns with the implemented operation (which is in descending order).

**Review by dlpbc — APPROVED** [review]
Other than the above clarification about the comment, which is really minor thing, the update looks good to me.

File: rctab/routers/accounting/routes.py (hunk 1 above)
Comment (Iain-S) [reply rctab/routers/accounting/routes.py:450]: Good point. I don't think that the comment is true. Let me fix that.

**Comment by Iain-S** [comment]
Note I had to add an exception for [this](https://data.safetycli.com/vulnerabilities/CVE-2019-8341/70612/) vulnerability. It is level critical but I believe it only applies if you are accepting templates as strings from users.

**Pull request merged**